docs/src/content/docs/id/guides/customization.mdx

```diff
@@ -171,7 +180,7 @@ tableOfContents: false
   </TabItem>
   <TabItem label="Konfigurasi Global">
 
-```js
+```js {7}
```

Comment:
```suggestion
```js {8}
```

Due to formatting, the line with `tableOfContents` is 1 line below in this language.

Reply:
Ah i see, thank you for the feedback. I will fix it soon.